easybuild/easyblocks/p/pytorch.py

```diff
@@ -280,6 +280,28 @@ def test_step(self):
 
         tests_out, tests_ec = test_result
 
+        # Show failed subtests to aid in debugging failures
+        # I.e. patterns like
+        # === FAIL: test_add_scalar_relu (quantization.core.test_quantized_op.TestQuantizedOps) ===
+        # --- ERROR: test_all_to_all_group_cuda (__main__.TestDistBackendWithSpawn) ---
+        regex = r"^[=-]+\n(FAIL|ERROR): (test_.*?)\s\(.*\n[=-]+\n"
+        failed_test_cases = re.findall(regex, tests_out, re.M)
+        # And patterns like:
+        # FAILED test_ops_gradients.py::TestGradientsCPU::test_fn_grad_linalg_det_singular_cpu_complex128 - [snip]
+        regex = r"^(FAILED) \w+\.py.*::(test_.*?) - "
+        failed_test_cases.extend(re.findall(regex, tests_out, re.M))
+        if failed_test_cases:
+            errored_test_cases = sorted(m[1] for m in failed_test_cases if m[0] == 'ERROR')
+            failed_test_cases = sorted(m[1] for m in failed_test_cases if m[0] != 'ERROR')
+            msg = []
+            if errored_test_cases:
+                msg.append("Found %d individual tests that exited with an error: %s"
+                           % (len(errored_test_cases), ', '.join(errored_test_cases)))
+            if failed_test_cases:
+                msg.append("Found %d individual tests with failed assertions: %s"
+                           % (len(failed_test_cases), ', '.join(failed_test_cases)))
+            print("\n".join(msg))
```

Comment:
The `print("\n".join(msg))` should either be removed or changed to use `print_warning` (or `print_msg`).

Reply:
Ouch, this was supposed to be:
```suggestion
            self.log.warning(msg)
```

but accidentally changed in https://github.com/easybuilders/easybuild-easyblocks/pull/2983/commits/c2bde44a7f2c3f71a2a146c2f8e1b9acca1d68d4